content/systems/filecoin_token/payment_channels/_index.md

```diff
@@ -2,40 +2,136 @@
+}
+```
+
+Before continuing with the details of the Payment Channel and its components and features, it is worth defining a few terms.
+
+- Voucher: a signed message from the creator of the channel (sender) to the recipient that updates the channel balance.
+- Redeeming a voucher: the voucher is submitted on-chain by the channel's recipient, but does not trigger movement of funds from the channel to the recipient's account. This action does not incur transaction/gas costs.
+- `UpdateChannelState`: this is the process by which a voucher is redeemed, i.e., a voucher is submitted (but not cashed-out) on-chain.
+- `Settle`: this process starts closing the channel. It can be called by either the channel creator (sender) or the channel recipient.
+- `Collect`: with this process funds are eventually transferred from the sender to the recipient. This process incurs transaction/gas costs.
+
+Throughout the interaction between the two parties, the channel sender (`From` address) is sending vouchers to the recipient (`To` address). The `Value` indicated in the signed message, i.e., _voucher_, from the sender to the recipient is progressively increasing together with the `Nonce` to indicate the value available to the provider to _redeem_. The `Value` is based on the service that the recipient has provided to the sender of the channel. The recipient can `Update` the balance of the channel and the balance `ToSend` to them. The recipient can choose to `Collect` this balance at any time incurring the corresponding gas cost.
+
+## Vouchers
+
+Traditionally, in order to transact through a Payment Channel parties send to each other signed messages that update the balance of the channel. In Filecoin, these signed messages are called _vouchers_.
+
+Redeeming a voucher is not transferring funds from the payment channel to the recipient's account. Instead, redeeming a voucher denotes the fact that some service worth of `voucher_val` has been provided by the recipient of the payment channel to the sender. It is not until the voucher is _collected_ that the funds are dispatched to the provider's account.
+
+This is the structure of the voucher:
+
+```go
+// A voucher is sent by `From` to `To` off-chain in order to enable
+// `To` to redeem payments on-chain in the future
+type SignedVoucher struct {
+	// ChannelAddr is the address of the payment channel this signed voucher is valid for
+	ChannelAddr addr.Address
+	// TimeLockMin sets a min epoch before which the voucher cannot be redeemed
+	TimeLockMin abi.ChainEpoch
+	// TimeLockMax sets a max epoch beyond which the voucher cannot be redeemed
+	// TimeLockMax set to 0 means no timeout
+	TimeLockMax abi.ChainEpoch
+	// (optional) The SecretPreImage is used by `To` to validate
+	SecretPreimage []byte
+	// (optional) Extra can be specified by `From` to add a verification method to the voucher
+	Extra *ModVerifyParams
+	// Specifies which lane the Voucher merges into (will be created if does not exist)
```

Comment:
You're right it's lane merging, not voucher merging.
In any case it may be worth using a different word here as lane merging is a different thing from a voucher being added to a lane. A voucher with a higher nonce supersedes a voucher with a lower nonce in the same lane.